tools/buildroot_checksum.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/sh
+set -e
+write=0
+latest_commit_branch=""
```

Comment:
Maybe add comments here about what these variables are for?

Reply:
Added